lessons/Data-Science/AI/ai-training/langs/en-us/pages/human-judgment.adoc

```diff
@@ -0,0 +1,69 @@
+= The Importance of Human Judgment
+
+Open the @starter-file{plagiarism}, click "Run", and then respond to the questions below.
+
+== Determine Closeness
+
+@n Use `cosine-similarity` to compare the student-submitted essay with the wikipedia article. Click "Run". What does the program tell you about the comparison? @fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.97087...,}}
+
+@fitb{}{@ifsoln{ indicating that the texts are extremely close.}}
+
+@n Did you know that "pachyderm" is a synonym for "elephant"? In the student essay, replace all instances of "elephant" with "pachyderm". Try running the program again, to compare `student-essay` and `wiki-article`. What does the program tell you about the comparison?
+
+@fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.7961..., meaning that the texts are still}}
+
+@fitb{}{@ifsoln{close enough to raise suspicion.}}
+
+
```

Comment:
An exercise that students would get into is to have to change the essay themselves (perhaps we could provide the og text in the starter file as the only version of the essay in the definitions area and define is as  `my-essay`. Then ask students to make some changes and see how it fairs in the plagiarism detector. Perhaps we first want them to record their words on this page?